docs/README.md

```diff
@@ -1,58 +1,68 @@
-
-### Hooks
-
-Husky hooks tha runs on every `pre-commit` and `pre-push`.
-
-### Intl Equalizer
-
-Tool for equalizing the messages located on the `messages` folder/builder. It's configured to use the **en.json** as the default file for comparison. For references, visit our [repository](https://github.com/vtex/intl-equalizer).
-
-### Lint + Formatting
-
-TS lint configured with Prettier and .Config.
+```jsonc
+{
+  "vtex.store-link": "0.x"
+}
+```
 
-### Available Scripts
+2. Now you can use the blocks exported by the app. In the following example you can see the usage of the `link.product` inside of a quick view component:
 
-```json
+```jsonc
 {
-  "lint": "cd ./react && yarn lint",
-  "test": "cd ./react && yarn test",
-  "lint:locales": "intl-equalizer",
-  "locales:fix": "intl-equalizer --fix",
-  "verify": "yarn lint && yarn lint:locales && yarn test"
+  "link.product#product-page": {
+    "props": {
+      "href": "/{slug}/p",
+      "label": "Mais detalhes >"
+    }
+  },
+  "product-summary.shelf": {
+    "children": [
+      "product-summary-image",
+      "product-summary-name",
+      "product-rating-inline",
+      "product-summary-space",
+      "product-summary-price",
+      "link.product#product-page
+    ]
+  },
 }
 ```
 
-### Ci
+You can see that there is a `{slug}` placeholder being passed to the `href` prop. When rendered, the placeholder will be replaced with the respective value of the closest product context and generate a link like `/everyday-necessaire/p`. However, keep in mind that in order for this work, you have to place the `link.product` block inside of a another block that provides a product context, such as the [`ProductSummary`](https://vtex.io/docs/components/product/vtex.product-summary).
 
-#### Install:
+### Props
 
-```yml
-install:
-  commands:
-    - echo Installing Packages...
-    - cd react
-    - npm install
-    - echo Packages installed!
-```
+The `link.product` and `link` blocks has the same props.
 
-#### Pre-build:
+| Prop name | Type     | Description                             | Default value |
+| --------- | -------- | --------------------------------------- | ------------- |
+| `label`   | `string` | The label that you be shown to the user | `undefined`   |
+| `href`    | `string` | The actual link                         | `'#'`         |
 
-```yml
-pre_build:
-  commands:
-    - echo Running tests...
-    - npm run verify
-    - echo Lint and tests finished!
-```
+### Available variables
+
+#### link.product
+
+| Value          | Description                                                                                    |
+| -------------- | ---------------------------------------------------------------------------------------------- |
+| `'slug'`       | The slug of the product                                                                        |
```

Comment:
Product slug

Reply:
What does it mean?

Reply:
I believe it would be better if we give more details about what should be filled out in `slug`